pkg/workceptor/kubernetes.go

```diff
@@ -51,6 +52,7 @@ type kubeUnit struct {
 	config              *rest.Config
 	clientset           *kubernetes.Clientset
 	pod                 *corev1.Pod
+	PodPendingTimeout   time.Duration
```

Comment:
let's use camelCase here and elsewhere, so instead of pod_pending_timeout_string it should be podPendingTimeoutString

Reply:
I have made the required changes.